src/groovy/org/pih/warehouse/api/StockTransferItem.groovy

```diff
@@ -67,6 +67,7 @@ class StockTransferItem {
 
     static StockTransferItem createFromProductAvailability(ProductAvailability productAvailability) {
         StockTransferItem stockTransferItem = new StockTransferItem()
+        def picklistItemQuantityPicked = PicklistItem.findByInventoryItem(productAvailability.inventoryItem)?.quantity ?: 0
```

Comment:
I am not sure if we should consider something as picked if it is in on hold or recalled.